core/common/src/test/java/alluxio/security/authorization/PermissionTest.java

```diff
@@ -124,4 +124,57 @@ private void verifyPermission(String user, String group, short mode, Permission
     Assert.assertEquals(group, permission.getGroup());
     Assert.assertEquals(mode, permission.getMode().toShort());
   }
+
+  /**
+   * Tests the {@link Permission#getOwnerBits(short)} method.
+   */
+  @Test
+  public void getOwnerBitsTest() throws Exception {
+    final short[] inputs = new short[] {
+        0700,
+        0505,
+        0640,
+        0000,
+    };
+    final short[] results = new short[] {
+        0007,
+        0005,
+        0006,
+        0000,
+    };
+    for (int i = 0; i < inputs.length; i++) {
+      Assert.assertEquals(results[i], Permission.getOwnerBits(inputs[i]));
```

Comment:
Agreed, thanks.
but this section is removed for other reason, I've added a few new tests in `ModeTest`.